tools/c7n_azure/c7n_azure/query.py

```diff
@@ -197,4 +207,25 @@ def register_actions_and_filters(registry, _):
             klass.action_registry.register('notify', Notify)
 
 
+@six.add_metaclass(QueryMeta)
+class ChildResourceManager(QueryResourceManager):
+
+    ParentSpec = namedtuple("ParentSpec", ["manager_name", "annotate_parent"])
+
+    class resource_type(QueryResourceManager.resource_type):
+        parent_spec = None
+
+    child_source = 'describe-child-azure'
+
+    @property
+    def source_type(self):
+        source = self.data.get('source', self.child_source)
```

Comment:
nit: describe is the only common alias for source, we typically have a source registry here, for other names that the base class falls back to.